src/Workspaces/Remote/ServiceHub/ExternalAccess/UnitTesting/Api/UnitTestingRoslynServicesWrapper.cs

```diff
@@ -2,19 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.Remote;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
 {
+    [Obsolete]
     internal readonly struct UnitTestingRoslynServicesWrapper
     {
         private readonly SolutionService _solutionService;
 
         public UnitTestingRoslynServicesWrapper(UnitTestingPinnedSolutionInfoWrapper pinnedSolutionInfoWrapper, UnitTestingAssetStorageWrappper assetStorageWrappper)
             => _solutionService = new SolutionService(SolutionService.CreateAssetProvider(pinnedSolutionInfoWrapper.UnderlyingObject, assetStorageWrappper.UnderlyingObject));
 
+        public UnitTestingRoslynServicesWrapper(UnitTestingPinnedSolutionInfoWrapper pinnedSolutionInfoWrapper, UnitTestingAssetStorageWrapper assetStorageWrapper)
+            => _solutionService = new SolutionService(SolutionService.CreateAssetProvider(pinnedSolutionInfoWrapper.UnderlyingObject, assetStorageWrapper.UnderlyingObject));
```

Comment:
Dup?

Reply:
I feel you might not review the latest commit? This commit https://github.com/dotnet/roslyn/pull/43856/commits/0186c8210211a9c1b8b0d6be234cb0caea395650 remove it.
The above one it the old one we are using now(It has a typo in it)